index.html

```diff
@@ -45,5 +45,194 @@ <h1 class="title">I'm Mariane <br>Glad to see you!</h1>
                 
     </section>
 
+
+    <section class="works-container" id="works-container">
+        <div class="works">
+            <img src="images/Snapshoot Portfolio.svg" alt="" class="tonic">
+            <img src="images/Snapshoot Portfolio1.svg" alt="" class="tonic-1">
+            <div class="div1">
+                <h1 class="header-tonic">Tonic</h1>
+
```

Comment:
`HTML` header tags follow a hierarchy, from `<h1>` to `<h6>`.
- `H1` tags are used to denote the most important text, such as the main theme or title of a content.
- `H2` and `H3` tags are commonly used as subheadings.
- Finally, `H4`, `H5`, and `H6` tags may be used to provide further structure within those subsections.

- [ ] Please use `hx` tags (i.e. `h2`, `h3`, `h4`, `h5` , `h6`) instead of `h1` tag. 

_Note: keep in mind the hierarchy of these tags e.g.` h3` can't come before` h2 `and `h3 ` can't be placed after` h4` etc._

Reply:
Done